app/forms/questionnaire_form.py

```diff
@@ -337,7 +337,9 @@ def _get_period_range_for_single_date(
         )
 
         if not min_period_date or not max_period_date:
-            raise ValueError("Period range must have a start and end date")
+
+            invalid_period_range = "Period range must have a start and end date"
```

Comment:
I think we need to be consistent throughout if we're gonna make them constants 👍